### PR TITLE
Update zoom link for coworking (and details)

### PR DESCRIPTION
**Pull request by steffilazerte**

### CODE REVIEW

**Comment by steffilazerte** [comment]
@maelle, could you take a look at the event for March? Since I'm trying to introduce themes to each coworking, I'm updating the way the events are posted. What do you think? Any suggestions?

File: content/events/2022-03-01-coworking-2022-03.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Social Coworking and Office Hours
+title: Social Coworking and Office Hours - Changing maintainers
```
Comment (maelle) [inline content/events/2022-03-01-coworking-2022-03.md:2]:
```suggestion
title: Social Coworking and Office Hours - Changing package maintainers
```

File: content/events/2022-03-01-coworking-2022-03.md [hunk 2]
```diff
@@ -28,20 +29,29 @@ lubridate::with_tz(d, "UTC")
 ```
 -->
 
-**Join us for 2 hours Tuesday, March 1st, 9 AM North American Pacific / 17:00 UTC for Social Coworking + Office Hours.**
+**Join us for 2 hours Tuesday, March 1st, 9 AM North American Pacific / 17:00 UTC for<br>[Social Coworking + Office Hours](/blog/2021/08/17/coworking-sessions/)**
 
-We are hosting [Social Coworking + Office Hours](/blog/2021/08/17/coworking-sessions/) on the first Tuesday of each month, alternating times to accommodate different parts of the world.
+This March we're focusing on the theme: **Changing maintainers** with community host
```
Comment (maelle) [inline content/events/2022-03-01-coworking-2022-03.md:34]:
```suggestion
This March we're focusing on the theme: **Changing package maintainers** with community host
```

File: content/events/2022-03-01-coworking-2022-03.md [hunk 3]
```diff
@@ -28,20 +29,29 @@ lubridate::with_tz(d, "UTC")
...
+- Ensure your R packages are understandable to potential collaborators (or future maintainers)
 - Cowork independently on work related to R
   - Plan out that package you've always wanted to create
   - Work on packages that tend to be neglected
-  - Make your packages more [contributor-friendly](/blog/2021/04/28/commcall-pkg-community/) (create issues, write a road map, etc.)
   - What ever you need to get done!
+  
+### Socialize/Network...
+
+- Meet community host, **Hannah Owens**, and ask about her experience taking over maintenance of the rOpenSci spocc package
```
Comment (maelle) [inline content/events/2022-03-01-coworking-2022-03.md:49]:
```suggestion
- Meet community host, **Hannah Owens**, and ask about her experience taking over maintenance of the [rOpenSci spocc package](https://docs.ropensci.org/spocc/).
```

**Review by maelle — APPROVED** [review]
LGTM, made a few suggestions

File: content/events/2022-04-05-coworking-2022-04.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Social Coworking and Office Hours
+title: Social Coworking and Office Hours - Making figures sparkle
```
Comment (maelle) [inline content/events/2022-04-05-coworking-2022-04.md:2]: :sparkler:

**Pull request merged**